frontend/express/public/javascripts/countly/countly.helpers.js

```diff
@@ -3435,6 +3435,22 @@
             return period;
         }
     };
+    /**
+    * Format number to percentage value
+    * @memberof CountlyHelpers
+    * @param {number} value number to be converted to percentage    
+    * @param {number} decimalPlaces number of decimal places to keep for percentage, default is two
+    * @returns {number} percentage number for given value. Otherwise, returns 0 for falsy or non number values
+    */
+    CountlyHelpers.formatPercentage = function(value, decimalPlaces) {
+        if (isNaN(value) || !value) {
+            return 0;
+        }
+        if (!decimalPlaces) {
+            decimalPlaces = 2;
+        }
+        return parseFloat((Math.round(value * 100)).toFixed(decimalPlaces));
+    };
```

Comment:
hmmm... I was not able to find it. I had created one earlier in of the plugins, so I decided to move it to commonly used utilities. In case you see it, please let me know I can always re-use it.